cli.go

```diff
@@ -32,20 +51,20 @@ func main() {
 
 	var err error
 	if *installFlag {
-		_ = PromptDiscord("patch").patch()
+		PromptDiscord("patch", nil, *dir, *client).patch()
 	} else if *uninstallFlag {
-		_ = PromptDiscord("unpatch").unpatch()
+		PromptDiscord("unpatch", nil, *dir, *client).unpatch()
 	} else if *updateFlag {
-		_ = installLatestBuilds()
```

Comment:
why did you remove these `_ =`? They are there to explicitly discard the potential error returned by these functions and suppress lint errors

![image](https://github.com/Vencord/Installer/assets/45497981/2b4fd2df-d914-469f-bcef-72e755a1e005)